src/main/java/unicash/model/category/Category.java

```diff
@@ -24,7 +26,7 @@ public class Category {
     public Category(String category) {
         requireNonNull(category);
         checkArgument(isValidCategory(category), MESSAGE_CONSTRAINTS);
-        this.category = category;
+        this.category = category.toLowerCase();
```

Comment:
If we wanted to ever revert to case-sensitive categorizing later, all we would need to do is just remove this part

Reply:
Could leave a comment in the Category java docs that mentions all Category's are case insensitive?